### DISCUSSION TITLE
Arrow keys in 5250 terminal

### DISCUSSION THREAD

**Discussion by JonasEngstedt** [codefori/vscode-ibmi#1675] [Q&A]
Hello there!

I'm inexperienced when it comes to terminals as a whole; but I seem to have an issue when it comes to the 5250 integrated terminal in Code 4 i. Whenever I try to use my arrow keys to navigate the terminal it seems to type the escape characters ([D, [C, [A, [B) instead of actually moving my cursor. It seems to work sometimes but all of the sudden stops and reverts back to the escape characters instead. I've tried seaching through all the settings and keybinds but can't seem to find anything on my own. Is there a simple fix to this that I'm missing?(fingers crossed).

**Comment by JonasEngstedt** [comment]
Just got the answers from my colleagues who have been using code for i for a while and as I understood it this is not something that will be fixable.

**Reply by worksofliam** [reply]
Hey @JonasEngstedt 

Not that it won't be fixable, but something I cannot accurately recreate to be honest.

I'm on a Mac and have bash setup on my user profile. What's your setup like?

**Reply by JonasEngstedt** [reply]
Damn you reply fast, appreciate it. 

Honestly I have no idea, not what the setup is like nor how to see it. As I said I'm quite inexperienced with terminals as a whole. The only terminal I've actually worked with is the 5250 emulator on Access client solutions, so I'm really fumbling in the dark atm :P

**Reply by chrjorgensen** [reply]
@JonasEngstedt If you got ACS installed, I would recommend you stick to the 5250 terminal in ACS - much better experience!
